dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKUtilities.java

```diff
@@ -665,28 +665,47 @@ public static Properties updateWrapperPropertiesFile(File wrapperPropertiesFile,
      * @return {@code true} if we are configured to use artifactory, {@code false} if not
      */
     static boolean useArtifactory() {
-        String forceExternalString = System.getProperty(FAT_TEST_PREFIX + ARTIFACTORY_FORCE_EXTERNAL_KEY);
-        boolean forceExternal = Boolean.parseBoolean(forceExternalString);
-
+        boolean forceExternal = isArtifactoryForceExternal();
         String artifactoryServer = getArtifactoryServer();
-        boolean haveArtifactoryServer = (artifactoryServer != null && !artifactoryServer.isEmpty());
 
-        boolean useArtifactory = haveArtifactoryServer && !forceExternal;
+        boolean useArtifactory = (Objects.nonNull(artifactoryServer) && !artifactoryServer.isEmpty());
+
+        if (forceExternal && !useArtifactory) {
+            throw new IllegalStateException("Forced external artifactory usage, but no server was set");
+        }
```

Comment:
This change is wrong. The point of `forceExternal` is to force the use of external servers (i.e. maven central, as opposed to an internal artifactory server), even if an artifactory server address is configured.